app/services/dttp/update_trainee_status.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Dttp
+  class UpdateTraineeStatus
+    include ServicePattern
+
+    ENDPOINTS = {
+      contact: "/contacts",
+      placement_assignment: "/placementassignments",
+    }.freeze
```

Comment:
https://stackoverflow.com/questions/37799296/ruby-what-does-the-comment-frozen-string-literal-true-do

Reply:
This is a hash though. We still need to freeze those (if you're saying that we don't need the `.freeze`)